sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -24,6 +24,8 @@ import javax.xml.bind.DatatypeConverter
 
 import scala.annotation.tailrec
 
+import sun.util.calendar.CalendarSystem
```

Comment:
We should be using `java.util.Calendar` right -- I presume any `sun.` classes are not for use by applications.